tests/outputs/TestPar.cpp

```diff
@@ -35,31 +35,34 @@ TEST(TestPar, write) {
                                                                          GeneratorDefinition::ReactiveCurvePoint(1., 11., 110.),
                                                                          GeneratorDefinition::ReactiveCurvePoint(2., 22., 220.),
                                                                          GeneratorDefinition::ReactiveCurvePoint(3., 33., 330.),
                                                                          GeneratorDefinition::ReactiveCurvePoint(4., 44., 440.),
                                                                      },
-                                                                     2., 20., 22., 220.),
+                                                                     2., 20., 22., 220., 100.),
                                                  GeneratorDefinition("G2", GeneratorDefinition::ModelType::DIAGRAM_PQ_SIGNALN, "02",
                                                                      {
                                                                          GeneratorDefinition::ReactiveCurvePoint(1., 11., 110.),
                                                                          GeneratorDefinition::ReactiveCurvePoint(2., 22., 220.),
                                                                          GeneratorDefinition::ReactiveCurvePoint(3., 33., 330.),
                                                                          GeneratorDefinition::ReactiveCurvePoint(4., 44., 440.),
                                                                      },
-                                                                     3., 30., 33., 330.),
+                                                                     3., 30., 33., 330., 100.),
                                                  GeneratorDefinition("G3", GeneratorDefinition::ModelType::WITH_IMPEDANCE_DIAGRAM_PQ_SIGNALN, "03",
                                                                      {
                                                                          GeneratorDefinition::ReactiveCurvePoint(1., 11., 110.),
                                                                          GeneratorDefinition::ReactiveCurvePoint(2., 22., 220.),
                                                                          GeneratorDefinition::ReactiveCurvePoint(3., 33., 330.),
                                                                          GeneratorDefinition::ReactiveCurvePoint(4., 44., 440.),
                                                                      },
-                                                                     4., 40., 44., 440.)};
+                                                                     4., 40., 44., 440., 100.),
+                                                 GeneratorDefinition("G4", GeneratorDefinition::ModelType::DIAGRAM_PQ_SIGNALN, "04",
+                                                                     {},
```

Comment:
Can you also test the case of a generator with targetP = 0 and without a diagram ?
This way we could see that for generators without diagrams we now have two sets of parameters : one with KGover=1 and one with KGover=0 and that in the .dyd we have the proper par set for each generator

Reply:
Hum, I did not think of that, that means that for every constant set in the par (such as impSignalNGenerator or signalNGenerator) we will duplicate it with the only thing changing being the "generator_KGover" ?

Reply:
That's it yes !
We would have impSignalNGenerator, signalNGenerator (with KGover = 1) and impSignalNGeneratorFixedP, signalNGeneratorFixedP (with KGover = 0) (any other names welcome !)

Reply:
And in the .dyd each generator could refer to each of these four sets depending on its situation